### ISSUE TITLE
Unable to use Barcode Scanner

### ISSUE THREAD

**Issue by wreardan** [ckumar1/smart-fridge-android#33]
On my phone, Nexus S 4G, I am unable to get the barcode scanner to actually scan any items.  The interface just keeps zooming in and out.  I tried oreos, half-baked ice cream, and beef jerky.  The camera is fairly low quality but I have scanned other items in the past with it.

**Comment by corypgr** [comment]
Does this contradict the issue you filed in #20 ?
If not, can you try to reproduce the actions taken to crash the app and the actions used to get the barcode scanner to show? Have you installed a barcode scanner since filing the other issue?

If the interface you are seeing does not register that there is a barcode then it is unlikely we have any control over that.

**Comment by wreardan** [comment]
Yes I installed zxing and now the app does not crash, but it will still not scan in a barcode label.
